web/src/tippyjs.ts

```diff
@@ -84,10 +85,22 @@ export const topic_visibility_policy_tooltip_props = {
     onShow(instance: tippy.Instance) {
         const $elem = $(instance.reference);
         const current_visibility_policy_str = $elem.attr("data-tippy-content");
+        const should_render_privacy_icon = $elem
+            .attr("aria-label")
+            ?.startsWith("Notifications are based on your configuration for");
```

Comment:
This won't work for non-English users once these strings are translated; the `aria-label` element will be in that language.

If you want to have logic depend on something, put that into a CSS class.